src/VisualStudio/Core/Def/Implementation/EditAndContinue/VsENCRebuildableProjectImpl.cs

```diff
@@ -601,7 +601,7 @@ private void AddActiveStatements(Solution solution, ShellInterop.ENC_ACTIVE_STAT
 
                     // If the PDB is out of sync with the source we might get bad spans.
                     var sourceLines = source.Lines;
-                    if (lineSpan.End.Line >= sourceLines.Count || lineSpan.End.Character > sourceLines[sourceLines.Count - 1].EndIncludingLineBreak)
+                    if (lineSpan.End.Line >= sourceLines.Count || sourceLines.GetPosition(lineSpan.End) > sourceLines[sourceLines.Count - 1].EndIncludingLineBreak)
```

Comment:
@tmat, is this testable?  Can we generate a PDB and then operate on modified source in a unit test?